datacentral.py

```diff
@@ -302,15 +334,39 @@ def generate(offline=False,
                 log.warn("%s: No local cache, skipping." % name)
                 continue
             else:
-                log.info("%s: New repo, cloning." % name)
-                repo = git.Repo.clone_from(url, dir_name)
-                updated = True
+                if url.endswith(".git"):
+                    # Handle GIT Repository URL
+                    log.info("%s: New repo, cloning." % name)
+                    try:
+                        repo = git.Repo.clone_from(url, dir_name)
+                        # For faster checkouts, one file at a time:
+                        #repo = git.Repo.clone_from(url, dir_name, n=True, depth=1)
+                        #repo.git.checkout("HEAD", "datapackage.json")
```

Comment:
Hmm. It's a good point, but not a priority for us at Central de Dados since we don't regenerate sites that often. So deleting the commented lines could work best, maybe putting them on a GH issue so that we don't lose track of possible improvements -- I'll do that.